functests/5_latency_testing/latency_testing.go

```diff
@@ -0,0 +1,210 @@
+package __latency_testing
+
+import (
+	//"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	//tool to test
+	OSLAT       = "oslat"
+	CYCLICTEST  = "cyclictest"
+	HWLATDETECT = "hwlatdetect"
+	//Environment variables
+	LATENCY_TEST_DELAY         = "LATENCY_TEST_DELAY"
+	LATENCY_TEST_RUN           = "LATENCY_TEST_RUN"
+	LATENCY_TEST_RUNTIME       = "LATENCY_TEST_RUNTIME"
+	MAXIMUM_LATENCY            = "MAXIMUM_LATENCY"
+	OSLAT_MAXIMUM_LATENCY      = "OSLAT_MAXIMUM_LATENCY"
+	HWLATDETEC_MAXIMUM_LATENCY = "HWLATDETEC_MAXIMUM_LATENCY"
+	CYCLICTEST_MAXIMUM_LATENCY = "CYCLICTEST_MAXIMUM_LATENCY"
+	LATENCY_TEST_CPUS          = "LATENCY_TEST_CPUS"
+	//invalid values error messages
+	UNEXPECTED_ERROR = "Unexpected error"
+	//incorrect values error messages
+	INCORRECT_P1 = "the environment variable"
+	INCORRECT_P2 = "has incorrect value"
+
+	//success message
```

Comment:
success message _template_ (or regex? glob pattern?)

Reply:
regex , will note that .Thanks.